src/element/bounds.test.ts

```diff
@@ -1,9 +1,23 @@
-import { getElementAbsoluteCoords } from "./bounds";
-import { ExcalidrawElement } from "./types";
+import { getElementAbsoluteCoords, getElementBounds } from "./bounds";
+import { ExcalidrawElement, ExcalidrawLinearElement } from "./types";
 
-const _ce = ({ x, y, w, h }: { x: number; y: number; w: number; h: number }) =>
+const _ce = ({
+  x,
+  y,
+  w,
+  h,
+  a,
+  t,
+}: {
+  x: number;
+  y: number;
+  w: number;
+  h: number;
+  a?: number;
+  t?: string;
+}) =>
```

Comment:
Not related to this but I really don't understand the purpose of this function. It just makes reading the tests harder.

Reply:
I guess the purpose of this is to make the test body more readable.